src/containers/forgot-password/index.tsx

```diff
@@ -2,12 +2,20 @@ import React from 'react';
 import { Helmet } from 'react-helmet';
 import { Button, Form, Input, Typography } from 'antd';
 import authClient from '../../auth/authClient';
+import { ForgotPasswordRequest } from '../../auth/ducks/types';
 
 const { Title } = Typography;
 
 const ForgotPassword: React.FC = () => {
-  const onFinish = (values: any) => {
-    authClient.forgotPassword(values);
+  const onFinish = (values: ForgotPasswordRequest) => {
+    authClient
+      .forgotPassword(values)
+      .then(() => {
+        alert('Successfully sent forgot password request!');
```

Comment:
Same as the previous comment, would be cool to render something instead of an alert